inbox/events/microsoft/parse.py

```diff
@@ -287,72 +288,54 @@ def convert_msgraph_patterned_recurrence_to_ical_rrule(
     )
 
 
-class SynthetizedCanceledoccurrence:
```

Comment:
I've changed this to just work as MsGraphEvent, even though those are not events coming from Ms Graph API. It simplified implementation and caused less problems with MyPy.